### PR TITLE
[eclipse/xtext#1679]ported more code to java

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1679]ported more code to java
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xbase.testing/testdata/testdata/CloseException.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package testdata;
+
+public class CloseException extends Exception {
+	private static final long serialVersionUID = -1653160717226140056L;
+	private final String text = "Exception on close";
```
Comment (tivervac) [inline org.eclipse.xtext.xbase.testing/testdata/testdata/CloseException.java:13]: static final

File: org.eclipse.xtext.xbase.testing/testdata/testdata/TryBodyException.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package testdata;
+
+public class TryBodyException extends Exception {
+	private static final long serialVersionUID = 8226120203611844202L;
+	private final String text = "Exception in try (with resources) body";
```
Comment (tivervac) [inline org.eclipse.xtext.xbase.testing/testdata/testdata/TryBodyException.java:13]: static final

File: org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/XbaseQualifiedNameConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+public class XbaseQualifiedNameConverterTest {
+	@Inject
+	private XbaseQualifiedNameConverter converter;
+
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@Test
+	public void test_toQualifiedName() {
+		HashMap<List<String>, String> testdata = new HashMap<>();
```
Comment (tivervac) [inline org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/XbaseQualifiedNameConverterTest.java:43]: Map on left hand side

File: org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/imports/DefaultImportsConfigurationTest.java [hunk 4]
```diff
@@ -8,51 +8,42 @@
...
-      _builder.newLine();
-      final ImportSectionTestLanguageRoot root = this._parseHelper.parse(_builder.toString().replaceAll("\r\n", "\n"));
-      Resource _eResource = root.eResource();
-      Assert.assertEquals(19, this.importsConfiguration.getImportSectionOffset(((XtextResource) _eResource)));
-    } catch (Throwable _e) {
-      throw Exceptions.sneakyThrow(_e);
-    }
-  }
+	@Inject
+	@Extension
```
Comment (tivervac) [inline org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/imports/DefaultImportsConfigurationTest.java:32]: Useless in java

File: org.eclipse.xtext.xbase.testing/testdata/testdata/CloseException.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.xbase.testing/testdata/testdata/CloseException.java:13]: was like that before

File: org.eclipse.xtext.xbase.testing/testdata/testdata/TryBodyException.java (hunk 2 above)
Comment (cdietrich) [reply org.eclipse.xtext.xbase.testing/testdata/testdata/TryBodyException.java:13]: was like that befoe

Comment (tivervac) [reply org.eclipse.xtext.xbase.testing/testdata/testdata/TryBodyException.java:13]: You can safely improve it though ;)

**Pull request merged**